specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -4777,6 +4786,16 @@
         }
       },
       "description": "List of connection states snapshots."
+    },
+    "ConnectionMonitorEndpointLocationDetails": {
+      "type": "object",
+      "properties": {
+        "region": {
```

Comment:
>region [](http://example.com/codeflow?start=9&length=6)

Is this an azure location? Why not call it 'location'?

Reply:
ConnectionMonitorEndpointLocationDetails can also store availabilityZone along with region.

Reply:
how would that look ? 

              "locationDetails": {
                "region": "?"
              },

Reply:
Hi,
We need other details like 'availabilityZone' along with region name, that's the reason, we are using a ConnectionMonitorEndpointLocationDetails defination for locationDetails